tests/generator/test_routing.py

```diff
@@ -1423,4 +1423,164 @@ def test_default_metric_v6(self):
 method=auto
 ip6-privacy=0
 route-metric=5050
+'''})
+
+    def test_add_routes_to_different_tables_from_multiple_files(self):
```

Comment:
I think we should make some use of the `ND_*` templates from `generator/base.py` here, e.g. ND_DHCP or ND_VLAN, to avoid repeating ourselves too much. Maybe concatenate or str.replace()  relevant pieces of the string as needed.